task/04-arrays-tasks.js

```diff
@@ -415,7 +422,10 @@ function toStringList(arr) {
  *      { country: 'Russia',  city: 'Saint Petersburg' }
  */
 function sortCitiesArray(arr) {
-   throw new Error('Not implemented');
+   return arr.sort((a, b) => a.country > b.country ? 1 
+   : a.country == b.country
+   ? a.city > b.city ? 1 : -1
+   : -1);
 }
 
```

Comment:
avoid this code style, if-else is more readable than multiple conditional operators ? :